params/config.go

```diff
@@ -67,7 +67,10 @@ var (
 		ConstantinopleBlock: big.NewInt(7280000),
 		PetersburgBlock:     big.NewInt(7280000),
 		IstanbulBlock:       big.NewInt(9069000),
-		Ethash:              new(EthashConfig),
+		Istanbul: &IstanbulConfig{
+			Epoch:          30000,
```

Comment:
Nit: If it doesn't break anything, 17280 would be the more canonical default value.
```suggestion
			Epoch:          17280,
```